code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -496,3 +496,11 @@
 		return ..()
 	else
 		return FALSE
+
+	/datum/chemical_reaction/reagent_explosion/dimethylmercury
+		name = "Dimethylmercury explosion"
+		id = "dimethylmercury_explosion"
+		mix_message = "<span class='userdanger'>The chemicals vaporize instantly in a rush of heat!</span>"
+		required_reagents = list("dimethylmercury" = 1)
+		required_temp = 278
```

Comment:
As said above, this chemical WILL explode when taken by any mob under normal conditions.
If you don't think this is intended and you don't feel like writing down a specific check_special_react() for a different required temperature while the holder is a mob, you could merely turn the mob_react FALSE, but that'll lock out some of the !!fun!! with this grindy chemical.